server/model/monitor.js

```diff
@@ -595,7 +595,7 @@ class Monitor extends BeanModel {
                         let data = res.data;
 
                         // convert data to object
-                        if (typeof data === "string") {
+                        if (typeof data === "string" && res.headers["content-type"] !== "application/json") {
```

Comment:
I think a more reliable way would be to parse and on parse error assume that it is a string.
Opinions?

Reply:
try-catch added.